superset/utils/logging_configurator.py

```diff
@@ -41,19 +41,11 @@ def configure_logging(
         if app_config["SILENCE_FAB"]:
             logging.getLogger("flask_appbuilder").setLevel(logging.ERROR)
 
-        # configure superset app logger
-        superset_logger = logging.getLogger("superset")
-        if debug_mode:
-            superset_logger.setLevel(logging.DEBUG)
-        else:
-            # In production mode, add log handler to sys.stderr.
-            superset_logger.addHandler(logging.StreamHandler())
-            superset_logger.setLevel(logging.INFO)
-
-        logging.getLogger("pyhive.presto").setLevel(logging.INFO)
```

Comment:
This was only necessary because we had `DEBUG` as the default log level.

Reply:
@jessie-ross why is that? Any institution can override the default config.

Reply:
Hi @john-bodley, we've set the default log level to info here:
https://github.com/apache/superset/issues/21646

Marking this as resolved.